scripts/west_commands/runners/probe_rs.py

```diff
@@ -32,6 +32,10 @@ def __init__(self, cfg, chip,
             self.args += tool_opt
 
         self.elf_name = cfg.elf_file
+        if cfg.hex_file is not None:
+            self.hex_file = cfg.hex_file
+        else:
+            self.hex_file = None
```

Comment:
The `if` can be omitted. Most runners call this `hex_name`, so I would suggest naming this here `hex_name` as well for consistency.
```suggestion
        self.hex_name = cfg.hex_file
```